src/external/dataCenterGateway/index.ts

```diff
@@ -33,17 +33,16 @@ export const getAlignmentMetrics = async (runId: String) => {
   }
 
   const config = await getAppConfig();
-  const authClient = await createAuthClient();
   const url = config.datacenter.gatewayUrl;
   const graphQLClient = new GraphQLClient(url, {
     headers: {
-      authorization: `Bearer ${await authClient.getAuth()}`,
+      authorization: `Bearer ${await getEgoToken()}`,
     },
   });
   const query = ALIGNMENT_METRICS_BY_RUN_ID;
   const variables = { runId };
 
-  const data = await graphQLClient.request(query, variables).catch(err => {
+  const data = await graphQLClient.request(query, variables).catch((err: Error) => {
```

Comment:
this isn't 100% the proper way to do it but it should be safe for our purposes. as I understand it, there is no guarantee of the type of object thrown since JS allows dumb things like `throw "hello there"`. So all catch blocks should be:
```
catch(e) {
  // e could be anything
  if (e instanceof Error) {
    // Error handling
  } else {
    // Handle e by logging whatever non error type it is
  }
  
}
```